distributed/worker_state_machine.py

```diff
@@ -3244,6 +3264,33 @@ def _to_dict(self, *, exclude: Container[str] = ()) -> dict:
         info = {k: v for k, v in info.items() if k not in exclude}
         return recursive_to_dict(info, exclude=exclude)
 
+    @property
+    def task_counts(self) -> dict[TaskStateState | Literal["other"], int]:
+        # Actors can be in any state other than {fetch, flight, missing}
+        n_actors_in_memory = sum(
+            self.tasks[key].state == "memory" for key in self.actors
+        )
```

Comment:
This is O(n) but actors are such a niche feature that we should not care in 99% of the cases